src/netaddress.cpp

```diff
@@ -160,10 +145,6 @@ void CNetAddr::SetLegacyIPv6(Span<const uint8_t> ipv6)
         // IPv4-in-IPv6
         m_net = NET_IPV4;
         skip = sizeof(IPV4_IN_IPV6_PREFIX);
-    } else if (HasPrefix(ipv6, TORV2_IN_IPV6_PREFIX)) {
```

Comment:
Don't we need some kind of check here to be able to filter out torv2-in-ipv6 in old style `addr` messages?

Reply:
Yes, I thought they would be invalid (so ignored) without the removed code, but it may be better to be explicit about it with something like:
```cpp
    } else if (HasPrefix(ipv6, TORV2_IN_IPV6_PREFIX)) {
        // TORv2-in-IPv6 (unsupported). Unserialize as !IsValid(), thus ignoring them.
        // Mimic a default-constructed CNetAddr object which is !IsValid() and thus
        // will not be gossiped, but continue reading next addresses from the stream.
        m_net = NET_IPV6;
        m_addr.assign(ADDR_IPV6_SIZE, 0x0);
        return;
```

Reply:
done